server/slack-bot-manifest.yaml

```diff
@@ -8,8 +8,8 @@ display_information:
   background_color: "#1976d2"
```

Comment:
This file has nothing to do with adding readme to slackbot

Reply:
I checked slack bot setting/permissions for useless things few days ago. This file is important for creating bot, so I thought it was a good place to update that file